common/lib/xmodule/xmodule/js/spec/.gitignore

```diff
@@ -7,3 +7,4 @@
 !time_spec.js
 !collapsible_spec.js
 !xmodule_spec.js
+!sequence/display_spec.js
```

Comment:
I'm not sure why you want to Git ignore this file. I think that was only needed for CoffeeScript files so that the JS files didn't accidentally get checked in.